azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/implementation/ResourceManager.java

```diff
@@ -149,7 +149,8 @@ public ResourceManager withSubscription(String subscriptionId) {
     }
 
     private ResourceManager(RestClient restClient, String subscriptionId) {
-        super(restClient, subscriptionId);
+        super(null, subscriptionId);
```

Comment:
Was thinking of avoiding passing null, by having a ResourceBase::constructor overload that takes only subscription id and ResourceManager override ManagerBase::resourceManager()